app/views/videos/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%article.video
+  .container
+    .row
+      .video_large_cover.col-sm-7.col-sm-offset-1
+        %img{src: @video.cover_large_url}
```

Comment:
I would rework this to `= image_tag @video.cover_large_url` - it reads better compared to html.